test/resources/artifacts_pprinted

```diff
@@ -11,4 +11,5 @@
     [transport :as transport]]
    [org.httpkit.client :as http]
    [refactor-nrepl.externs :refer [add-dependencies]])
-  (:import java.util.Date))
+  (:import
```

Comment:
I made the treatment of :import homogeneous relative to that of :require, as most likely users never want an asymmetry.

(It's also what https://github.com/bbatsov/clojure-style-guide/tree/e6cb6dab0925bcfabcd4ec96a2f69f41bf9c698e#line-break-ns-declaration suggests in its example)